lti_consumer/tests/unit/test_lti_consumer.py

```diff
@@ -1,914 +1,367 @@
+# -*- coding: utf-8 -*-
```

Comment:
Don't worry about fixing tests before getting a working version of the code.
Otherwise you'll need to rework the code structure and the tests every time a review comes up.
For the early iterations of code, only change the main code and related libraries, and we can polish the implementation once it's already working and in it's final form.